### PR TITLE
Fix: Pricing table divider on small screens

### PR DESCRIPTION
**Pull request by gmjuhasz**
Fixes small bug introduced in #26253

#### Changes proposed in this Pull Request:

This is a quick PR to fix a bug with the picing-table design, where the divider between rows is not appearing on small screens

- Added `left` and `right` props to retain the width of the divider

#### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?

#### Jetpack product discussion

p1664203272317609-slack-C02JJ910CNL

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

* Fire up storybook with running `npm run storybook:dev` from the `js-packages/storybook` folder.
* Divider should be present on small screens as well

### CODE REVIEW

File: projects/js-packages/components/components/pricing-table/styles.module.scss [hunk 1]
```diff
@@ -75,6 +75,8 @@
 			content: '';
 			position: absolute;
 			top: 0;
+			left: var(--padding);
+			right: var(--padding);
```
Comment (danielpost) [inline projects/js-packages/components/components/pricing-table/styles.module.scss:79]: I think this actually breaks the lines on larger screens (pushes them to the right), so maybe we should remove these properties on large viewports?

**Review by danielpost — CHANGES_REQUESTED** [review]

File: projects/js-packages/components/components/pricing-table/styles.module.scss (hunk 1 above)
Comment (gmjuhasz) [reply projects/js-packages/components/components/pricing-table/styles.module.scss:79]: Yeah, good point, I meant to put there. Updated!

**Review by danielpost — APPROVED** [review]

**Review by pablinos — APPROVED** [review]
Re-approving

**Comment by spsiddarthan** [comment]
r253248-wpcom

**Pull request merged**